### PR TITLE
Update 08-annotation.Rmd

### PR DESCRIPTION
**Pull request by ehumph**

### CODE REVIEW

File: 08-annotation.Rmd [hunk 1]
```diff
@@ -75,11 +75,11 @@ install.packages("pheatmap")
 
 ```{r, warning = FALSE, message = FALSE}
 # AnVIL::install("pheatmap")
-#library(pheatmap)
+library(pheatmap)
```
Comment (avahoffman) [inline 08-annotation.Rmd:80]: do we need to add this to the docker image?

**Review by avahoffman — APPROVED** [review]

**Review by avahoffman — CHANGES_REQUESTED** [review]
Something funky happening in the build..

```

Quitting from lines 50-54 (08-annotation.Rmd) 
Error in .absent_action(x, val = output, fun = "colLabels", onAbsence = onAbsence) : 
  'colLabels(<SingleCellExperiment>)' returns 'NULL'
Calls: local ... .setup_groups -> colLabels -> colLabels -> .local -> .absent_action

Execution halted
Error in Rscript_render(f, render_args, render_meta, add1, add2) : 
  Failed to compile 08-annotation.Rmd
Calls: <Anonymous> -> render_new_session -> Rscript_render
Execution halted
```

File: 08-annotation.Rmd (hunk 1 above)
Comment (ehumph) [reply 08-annotation.Rmd:80]: It would probably be nice.

**Comment by ehumph** [comment]
> Something funky happening in the build..
> 
> ```
> 
> Quitting from lines 50-54 (08-annotation.Rmd) 
> Error in .absent_action(x, val = output, fun = "colLabels", onAbsence = onAbsence) : 
>   'colLabels(<SingleCellExperiment>)' returns 'NULL'
> Calls: local ... .setup_groups -> colLabels -> colLabels -> .local -> .absent_action
> 
> Execution halted
> Error in Rscript_render(f, render_args, render_meta, add1, add2) : 
>   Failed to compile 08-annotation.Rmd
> Calls: <Anonymous> -> render_new_session -> Rscript_render
> Execution halted
> ```

File: 08-annotation.Rmd (hunk 1 above)
Comment (ehumph) [reply 08-annotation.Rmd:80]: Though the code actually works without it (and with the local installation of pheatmap)

**Pull request merged**